README.md

```diff
@@ -1,2 +1,3 @@
-This is the first workshop about API testing by Manuel Osorio
+# workshop-api-testing-js
+## This is the first workshop about API testing by Manuel Osorio
 # workshop-api-testing-js
```

Comment:
Remove this line

Reply:
@Maosorio92 As Veronica says, the line I'm referring to is the third line, that has `# workshop-api-testing-js` repeated.